model/reactions.tsv

```diff
@@ -9868,7 +9868,7 @@ rxns	rxnKEGGID	rxnBiGGID	rxnEHMNID	rxnHepatoNET1ID	rxnREACTOMEID	rxnRecon3DID	rx
 "MAR09924"	""	"EX_stcrn_e"	""	""	""	"EX_stcrn[e]"	""	""	""	""	0	""	""	"EX_stcrn[e]"
 "MAR09925"	""	"EX_pmtcrn_e"	""	""	""	"EX_pmtcrn[e]"	""	""	""	""	0	""	""	"EX_pmtcrn[e]"
 "MAR09926"	""	"EX_hdcecrn_e"	""	""	""	"EX_hdcecrn[e]"	""	""	""	""	0	""	""	"EX_hdcecrn[e]"
-"MAR09927"	""	"PCREATte"	""	""	""	"PCREATte"	""	""	""	""	0	""	""	"PCREATte"
+"MAR09927"	""	"PCREATte"	""	""	""	"PCREATte"	""	""	""	""	1	""	""	"PCREATte"
```

Comment:
Are you sure that creatine phosphate can diffuse out of the cell? I have not seen any evidence of this. It would probably not be desirable for muscle cells to lose it, I don't know how this works. My comment was for the mitochondrial membrane only, that is well known and part of the creatine phosphate shuttle.

Reply:
Ok, then maybe to be on the safe side, we only mark the mitochondria -> cytosol reaction as spontaneous for now?

Reply:
That would be my recommendation